go/extra/ba/README.md

```diff
@@ -0,0 +1,61 @@
+# Benchmark analysis
```

Comment:
Add `Requirements` section, which seems to be a Prometheus instance listening on `localhost:9090`